package.json

```diff
@@ -11,9 +11,9 @@
 		"umd"
 	],
 	"scripts": {
-		"build": "nwb build-react-component",
+		"build": "NODE_ENV=production nwb build-react-component",
```

Comment:
this should have been implicit seeing as its a productions build are you sure this is necessary? did you read the nwb docs?

Reply:
How does `nwb` know its a production build? I've looked at the docs, and they don't specify this, unless it was right under my nose, and I didn't see it.